### PR TITLE
Palo alto

### PR DESCRIPTION
**Pull request by kgiori**
added resource links for Palo Alto

### CODE REVIEW

**Review by nancyhang — CHANGES_REQUESTED** [review]
Thanks! Kathy, can you please put this under San Mateo County before approving to be merged to master?

**Comment by nancyhang** [comment]
Actually, I mean Santa Clara county section, not San Mateo county.

File: README.md [hunk 1]
```diff
@@ -130,6 +130,31 @@ Then, you can remove the Bay Area specific information and add info particular t
...
+
+Go [here](https://www.cityofpaloalto.org/services/public_safety/plans_and_information/coronavirus/default.asp) for the City’s dedicated Coronavirus webpage.
+
+Go [here](https://us7.campaign-archive.com/home/?u=5bb3b762047926b143fbbee55&id=d8eda03e5f) for Daily Coronavirus Reports and sign up.
+
+Go [here](https://www.cityofpaloalto.org/services/public_safety/plans_and_information/coronavirus/support_services.asp) to get connected with Palo Alto community based organizations, resources and ways to volunteer and help.
+
+Go [here](https://www.cdc.gov/coronavirus/2019-ncov/faq.html) for the Centers for Disease Control and Prevention Frequently Asked Questions.
+
+Go [here](https://www.cdph.ca.gov/Programs/CID/DCDC/Pages/Immunization/ncov2019.aspx) for the State of California Department of Public Health.
```
Comment (ssami) [inline README.md]: Things like this website, CDC and links that aren't specific to Palo Alto can actually be put up at the top of the readme. Thanks for adding those links!

**Comment by kgiori** [comment]
the added palo alto section was listed under Santa Clara county, so I'll merge it and see what happens. :)

**Pull request merged**